src/content/docs/journey-app-slow/create-benchmarks.mdx

```diff
@@ -0,0 +1,171 @@
+* Have alerts set up to warn of any SLI or SLO breaches so you can react before they impact your users.
+
+## Why create performance benchmarks
+
+Now that you've remdiated the root cause of your app's slowness, let's set up service levels so you can respond quickly if any slowness appears again. 
+
+Let's look at the difference between service levels, SLOs and SLIs. Service levels describe services provided to users within a given period of time, in measurable terms. Service level objectives (SLOs) are the goals set for the availability expected out of a system. Service level indicators (SLIs) are the key measurements and metrics to determine the availability of a system. 
+
+Defining and managing your service levels following the steps below provides the ability to: 
+
+* Ease future setup: Automatically establish a baseline of performance and reliability for any service with a one-click setup.
+* Define reliability across teams: Avoid arduous alignment processes with SLO and SLI recommendations that help you determine service boundaries. Set reliability benchmarks automatically based on recent performance metrics in any entity.
+* Iterate and improve: With full-stack context and automation through open-source infrastructure-as-code tools like Terraform, teams have insight into how specific nodes or services impact system reliability and can quickly take control over their performance. Custom views for both service owners and business leaders drive operational efficiency and lead to better reporting, alerting, and incident management processes.
+* Standardize reliability: Cross-organizational teams have a unified, transparent view of service reliability, and can better comply with customer-facing SLAs. SLO compliance metrics and error budgets give organizations a way to report on reliability and implement changes across applications, infrastructure, and teams in a cohesive fashion.
+
+## Create performance benchmarks
+
+Let's get started with creating your front-end performance benchmarks:
+
+<Steps>
+    <Step>
+
+    ## Select service level indicators:
+
+    While there are a large amount of SLIs you could use to define your front end experience benchmarks, the following are some we specifically reccomend. Each collapser has an explanation of when you should choose that SLI and a corresponding NRQL query (which you'll use in step 2).
+
+    For now, just select one of the following:
+
+    **SLIs for APM services instrumented with the New Relic agent**:
+    Based on `Transaction` events, these SLIs are the most common for request-driven services:
+
+    <CollapserGroup>
+    <Collapser
+        className="freq-link"
+        id="service-availability"
+        title="Service success"
+    >
+        Service success is the ratio of the number of successful responses to the number of all requests. Effectively, this is an error rate, but you can filter it down, for example removing expected errors.
```

Comment:
The second sentence is a little unclear. Why would you want to filter down and what does the error rate tell about the service? I imagine it's probably variable depending on the service itself, but maybe providing a throw away example of what this looks like in action would elevate it as an ~example